contributing/DEVELOPING.md

```diff
@@ -47,6 +47,7 @@ The Quick Start Guide's  [One-time setup](getting-started-quick.md#one-time-setu
 | `gulp lint`                                                             | Validates against Google Closure Linter.                              |
 | `gulp lint --watch`                                                     | Watches for changes in files, Validates against Google Closure Linter.|
 | `gulp lint --fix`                                                       | Fixes simple lint warnings/errors automatically.                      |
+| `gulp lint --file=path/to/file.js`                                      | Lints just a single file.                                             |
```

Comment:
Per above comment, change this to:
```
gulp lint --files=<files-path-glob>
```
and
```
Lints just the files provided.
```

Reply:
done